HomeWork_One/Quiz_Template/src/components/inputs/Select.tsx

```diff
@@ -0,0 +1,25 @@
+import './Select.css'
+import { v4 as uuidv4 } from 'uuid'
+
+function Select({category,options}) {
+    console.log(options)
+    return(
```

Comment:
I think if there are no options (`options? option`) we may do not render this component
wdyt?